specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/virtualWan.json

```diff
@@ -2696,6 +2696,63 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/p2svpnGateways/{gatewayName}/reset": {
```

Comment:
what does this operation do?  Its not clear from the PR description or the description in the spec.  How does a user know when they need to do this and does it cause any downstream changes on any resource?

Reply:
Its used for rebooting customer P2SVpnGateway resource instance in case they need to do it for some debugging purpose. We have similar operations for existing resources:- VpnGateway, VirtualNetworkGateway.